sdk/schemaregistry/schema-registry/src/generated/schemaRegistryClientContext.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import * as coreHttp from "@azure/core-http";
+import { SchemaRegistryClientOptionalParams } from "./models";
+
+const packageName = "@azure/schema-registry";
+const packageVersion = "1.0.0";
```

Comment:
This needs to match the package version in package.json file.

```suggestion
const packageVersion = "1.0.0-preview.1";
```